dmscripts/models/queries.py

```diff
@@ -42,6 +42,15 @@ def process_fields(rules, data):
     return data
 
 
+def rename_fields(rules, data):
+    """ Return data after renaming any keys using the renaming rules """
+    for key in data.keys():
+        if key in rules.keys():
+            data[rules[key]] = data.pop(key)
+
+    return data
+
```

Comment:
I see what you mean about the `rules` naming, it does feel a little ambigious. Can you think of a potentially better name? We could use `columns` like pandas do?(http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.rename.html)

I didn't realise that about not needing to call `.keys()`. That's good to know! It does seem a little weird but at the same time makes sense (although feels a little less readable/explicit if you are a python noob like me)